central/nodecomponentcveedge/datastoretest/nodecomponentcveedge_from_componentcveedge_datastore.go

```diff
@@ -0,0 +1,60 @@
+package datastoretest
```

Comment:
If it's used only in `datastore` package then maybe better is to put it in `internal` package or `_test` file